### ISSUE TITLE
part-of relation tree expansion

### ISSUE THREAD

**Issue by micheldumontier** [bio-ontology-research-group/aberowl-orepo#94]
have a look at the new OLS for "hard palate" 
http://www.ebi.ac.uk/ols/beta/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0003216

http://aber-owl.net/ontology/HP#!http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0003216

**Comment by leechuck** [comment]
Yes, we are working on this; they only use the structural reasoner to generate this, we'll use Elk, which is a bit more difficult.
